handsontable/src/renderers/checkboxRenderer/__tests__/checkboxRenderer.spec.js

```diff
@@ -151,7 +151,7 @@ describe('CheckboxRenderer', () => {
     //  spec().$container.simulate('keydown',{
     //    keyCode: 32
     //  });
-    keyDown('space');
+    keyDownUp(' ');
```

Comment:
I would leave the name `Space` as the `' '` at first glance looks like some refactor leftovers